### DISCUSSION TITLE
Networks page scrambled

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#91] [Fixed Issues] [answered]
@rern 
I applied the 20200107 update this morning. There is  an error on the Networks page which prevents the page from being displayed:

Errors: Unexpected token } in JSON at position 380
{ "bluetooth" : false , "list" : [{ "dhcp" : "static" , "gateway" : "192.168.86.1" , "hostname" : "" , "interface": "eth0" , "ip" : "" , "mac" : "b8:27:eb:24:d8:7d" , "ssid" : "" , "dbm" : 0 },{ "dhcp" : "dhcp" , "gateway" : "192.168.86.1" , "hostname" : "RAConservatory.local" , "interface": "wlan0" , "ip" : "192.168.86.33" , "mac" : "b8:27:eb:71:8d:28" , "ssid" : "" , "dbm" : ▮}] , "hostapd" : {} , "hostname" : "RAConservatory" , "profiles" : ["CMHomeNET"] , "reboot" : "" , "wlan" : true}

Chris

**Comment by ceejayemm** [comment, marked as answer]
Fixed with Update 20201208

Thanks

Chris

**Comment by ANILkumarsinghII** [comment]
Big a** small
